app/src/fdroid/java/tice/managers/MapboxGeocodingManager.kt

```diff
@@ -0,0 +1,53 @@
+package tice.managers
+
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.JsonObject
+import kotlinx.serialization.json.jsonArray
+import kotlinx.serialization.json.jsonObject
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import tice.dagger.scopes.AppScope
+import tice.models.Coordinates
+import tice.utility.getLogger
+import java.util.*
+import javax.inject.Inject
+import javax.inject.Named
+
+@AppScope
+class MapboxGeocodingManager @Inject constructor(
+    private val okHttpClient: OkHttpClient,
+    @Named("MAPBOX_ACCESS_TOKEN") private val mapboxAccessToken: String
+) : MapboxGeocodingManagerType {
+    private val logger by getLogger()
+
+    private val baseURL = "https://api.mapbox.com/"
+
+    override suspend fun reverseGeocoding(coordinates: Coordinates): String? {
+        val urlString = baseURL +
```

Comment:
We could use a better url building here instead of concatting. Maybe https://developer.android.com/reference/kotlin/android/net/Uri.Builder ?